portal/eproms/templates/eproms/portal.html

```diff
@@ -65,7 +65,13 @@ <h2 class="tnth-subhead">{{ _("Welcome to TrueNTH") }}</h2>
                 </div>
             {%- else -%}
                 <div class="button-container">
-                    <a class="btn-lg btn-tnth-primary" href="{{ display.link_url }}">{{ display.link_label }}</a>
+                    <a class="btn-lg btn-tnth-primary" href="{{ display.link_url }}">
+                    	{%-if display.link_label %}
```

Comment:
You might be able to condense this into one line with the [jinja `default` filter](http://jinja.pocoo.org/docs/2.10/templates/#default). 
I think you'll have to pass `True` since `display.link_label` is defined, but evaluates to `False` (see link for more details)
```
{{ display.link_label|default(_("Go to link"), true) }}
```
I think this might work too...
```
{{ display.link_label or _("Go to link") }}
```